docs/deploy-ota.md

```diff
@@ -0,0 +1,62 @@
+# OTA Deployments
+
+## Overview
+
+![OTA Deployment](./img/ota-flow.png)
+
+## Internal Deployments
+
+Internal OTA deployments should be performed automatically upon all merges into main. In cases where the fingerprint diff results in incompatible native changes, a new client build will automatically be ran and deployed to TestFlight (iOS) or delivered in Slack (Android).
+
+## Production Deployments
+
+### Prerequisites
+
+- Remove any internal client from your device and download the client from the App Store/Google Play. This will help for testing as well as retrieving the build number.
+- You should have signed in to EAS locally through npx eas login. You will need to modify the build number in a subsequent step.
+- Identify the build number of the production app you want to deploy an update for. iOS and Android build numbers are divergent, so you will need to find both
+
+  ![app-build-number](./img/app-build-number.jpg)
+
+- Ensure that the commit the initial client was cut from is properly tagged. The tag should be in the format of 1.X.0
+
+### Preparation
+
+- Create a new branch from the tag that the initial release was cut from if no OTA deployment has been made yet for this client. Name this branch 1.X.0-ota-1
+- If a deployment has been made previously for this release, increment the tag, i.e. 1.x.0-ota-2
```

Comment:
Should pry clarify, you mean literal git tags here. So we have a branch naming convention, plus we also tag those commits with tags following the same convention?

Reply:
What purpose to tags serve here? When do we normally tag commits for release?

Reply:
There's no need to tag the commits on those other branches, since we would not see them on the main branch anyway. But yea, I'll clarify that.

The tags serve the purpose of knowing which commit to fingerprint your proposed deployment commit against. The workflow will get whatever commit hash was tagged with your input version and use that for the comparison.

Reply:
That last line should have been "increment the branch name". Changed.